### ISSUE TITLE
MessageTokenizer crash on Windows style end of line.

### ISSUE THREAD

**Issue by Merichbier** [siemens/ros-sharp#241]
* My OS is: Windows 10
* My Unity Version is: Unity 2019.2.0f1
* My ROS Distribution is: not relevant
* My Build Target Platform is: not relevant

**Here is my bug description:**
When trying to auto generate classes for Message/Service/Action, the tokenizer fail.
I tried the unity tool and the console tool, both fail.

Perform the following steps reproduce the bug:
- download [moveit messages definitions](https://github.com/ros-planning/moveit_msgs)
- try to autogenerate any files from the previous git repo
- Failed with message `Invalid character in identifier:  `

I also tried with the following simple msg file:
```
string first_name
string last_name
uint8 age
uint32 score
```
Behavior is the same

Observed results:
The character is actually a windows style line feed `\r`

Expected results:
The file should be able to be converted into a C# class

Relevant Code:
`MessageTokenizer.cs`
`line 119`
``` C#
if (reader.Peek() == '\n')
```
`line 315`
``` C#
while (reader.Peek() != ' ' && reader.Peek() != '\n' && reader.Peek() != '=' && !reader.EndOfStream)
```

Those two lines (as example) only check for linux style end of line. __We__ should create a function that control if the character is an end of line, checking both linux and Windows

**Comment by sye8** [comment]
- Couldn't reproduce issue from directly clones MoveIt messages
  - Manually edited with Notepad
- Attempted fix at [my fork](https://github.com/sye8/ros-sharp). Please test.

Thank you!

P.S: Consider getting [Sublime](https://www.sublimetext.com) or [Atom](https://atom.io/). They are awesome

**Comment by MartinBischoff** [comment]
resolved in commit #244

**Comment by Merichbier** [comment]
Sorry didn't had time to check, but it should work thanks !
